src/Shared/Components/CICDHistory/LogsRenderer.tsx

```diff
@@ -428,6 +445,18 @@ export const LogsRenderer = ({
                                 initialSearchText={searchKey}
                                 size={ComponentSizeType.large}
                             />
+                            <button
```

Comment:
Please use Button component

Reply:
consulted with product team and eshank regarding usage of button component here; since we don't have a white variant of the button component I was told to just stick with the normal button here